mintapi/signIn.py

```diff
@@ -352,7 +352,7 @@ def sign_in(
         expected_conditions.presence_of_element_located(
             (
                 By.CSS_SELECTOR,
-                "#ius-link-use-a-different-id-known-device, #ius-userid, #ius-identifier, #ius-option-username",
+                "ius-hosted-ui-main-container",
```

Comment:
Given that there's a good chance not all users have this new selector, I think we should just append to the existing list instead of starting a new one with just this one value.

Reply:
Thanks. I updated the CSS Selector to support the old CSS Class Names. However, for the new one to work for me, I had to use ".ius-hosted-ui-main-container" rather than "#ius-hosted-ui-main-container". I'm unsure if that's how the class names selectors worked previously.